client/src/GlobalStyles.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+import Roboto from "./fonts/roboto-regular.ttf";
+import Lora from "./fonts/lora-mediumItalic.ttf";
+
+const GlobalStyles = () => {
+  return (
+    <Global
+      styles={css`
+        *,
+        *:before,
+        *:after {
+          box-sizing: border-box;
+          margin: 0;
+        }
+        :root {
+          height: 100vh;
+          width: 100vw;
+          --font-color-dark: #383636;
+          --font-color-light: #f5f6fa;
+          --base-bg-color: linear-gradient(
+            to top,
+            #d5d4d0 0%,
+            #d5d4d0 1%,
+            #eeeeec 31%,
+            #efeeec 75%,
+            #e9e9e7 100%
+          );
+          --menu-bg-color: #cebebe;
+          --header-bg-color: #a79292;
+          --bubble-bg-color: #cebebe;
+        }
+
+        @font-face {
+          font-family: "Roboto";
+          src: url(${Roboto}) format("truetype");
+        }
+
+        @font-face {
+          font-family: "Lora";
+          src: url(${Lora}) format("truetype");
+        }
+
```

Comment:
You could also save the font-families as global variables in your root:

--font-lora: "Lora";
--font-roboto: "Roboto";

This way later you do not have to write ```serif```.

Reply:
Clean structure, very good, Sasha! @manuelsanchez2 's advice is good!